src/components/SettingsMenu.js

```diff
@@ -0,0 +1,283 @@
+                  </div>
+                </div>
+                <br />
+                <button
+                  className={styles.OKButton}
+                  onClick={() => {
+                    this.props.closeSettingsMenu();
+                  }}
+                >
+                  OK
+                </button>
+              </div>
+            ) : null}
+            {settingsMenuId === 1 ? (
+              <div style={{ padding: 20 }}>
+                <h4 style={{ padding: 0, margin: 0 }}>Map settings</h4>
+                <hr />
+                <div className={styles.MapSettings}>
+                  <p>
+                    There are {MAP_BACKGROUNDS
+                      ? MAP_BACKGROUNDS.length
+                      : 0}{" "}
+                    available map background(s):
+                    {MAP_BACKGROUNDS[0].description} and{" "}
+                    {MAP_BACKGROUNDS[1].description}.
+                  </p>
+                  <p>
+                    Currently selected:&nbsp;
+                    <strong>
+                      {this.props.currentMapBackground.description}
+                    </strong>.
+                  </p>
+                  <button onClick={this.toggleMapBackground.bind(this)}>
+                    Switch
+                  </button>
+                </div>
+                <br />
+                <button
+                  className={styles.OKButton}
+                  onClick={() => this.props.closeSettingsMenu()}
+                >
+                  OK
+                </button>
+              </div>
+            ) : null}
+
+            {settingsMenuId === 2 ? (
+              <div style={{ padding: 20 }}>
+                <h4 style={{ padding: 0, margin: 0 }}>Contact info</h4>
+                <hr />
+                <p>
+                  For software issues with the FloodSmart Parramatta System
+                  please contact Nelen & Schuurmans on {nensMail()}. For any
+                  other issues, or suggestions for improvements to the
+                  Parramatta Floodsmart System system, please contact Chris
+                  Gooch on tel.&nbsp;
+                  {chrisTel()} or email {chrisMail()}
+                </p>
+              </div>
+            ) : null}
+            {settingsMenuId === 3 ? (
+              <div style={{ padding: 20 }}>
+                <h4 style={{ padding: 0, margin: 0 }}>
+                  Choose another dashboard for training
+                </h4>
+                <hr />
+                <p>Choose training dashboard</p>
+                <select
+                  onChange={e => {
+                    const value = e.target.value;
+                    const url = "/floodsmart/" + value;
+                    window.location.href = url;
+                  }}
+                >
+                  <option disabled selected value>
+                    {" "}
+                    -- select an option --{" "}
+                  </option>
```

Comment:
Options showing all trainings client_configs.
These are coming from the redux store which in turn gets them from the client config where they are hardcoded.